README.md

```diff
@@ -41,38 +49,13 @@ $ npm run lint
 $ npm test
 ```
 
-To run the integration tests, they must be invoked independent of the unit tests:
```

Comment:
Why drop the integration and CI sections from the README? Are they no longer relevant?

Reply:
Added back the CI section but since we no longer have a working integration test (see issue #222 for adding a new Pipeline integration test) I thought removing the text on integration test would be less confusing and more accurate given the current state of the code-base.

Reply:
Ah, makes sense. Thanks!